apps/prairielearn/src/pages/studentAssessment/studentAssessment.ts

```diff
@@ -19,13 +19,19 @@ import {
   canUserAssignGroupRoles,
 } from '../../lib/groups.js';
 import { getClientFingerprintId } from '../../middlewares/clientFingerprint.js';
-import { checkPasswordOrRedirect } from '../../middlewares/studentAssessmentAccess.js';
+import logPageView from '../../middlewares/logPageView.js';
+import selectAndAuthzAssessment from '../../middlewares/selectAndAuthzAssessment.js';
+import studentAssessmentAccess, {
+  checkPasswordOrRedirect,
+} from '../../middlewares/studentAssessmentAccess.js';
 
 import { StudentAssessment } from './studentAssessment.html.js';
 
-const router = Router();
+const router = Router({ mergeParams: true });
 const sql = loadSqlEquiv(import.meta.url);
 
+router.use([selectAndAuthzAssessment, studentAssessmentAccess, logPageView('studentAssessment')]);
```

Comment:
This is a small nit, but IMO it'd be better to use sequential calls to `router.use`, each with a single middleware, rather than passing a full array. This may just come down to how Prettier formats this all onto one line, but I like to read top-to-bottom to see what happens.

Reply:
Changed these (and elsewhere) to sequential calls of `router.use` in [15771e5](https://github.com/PrairieLearn/PrairieLearn/pull/10906/commits/15771e55e6dd9050a1d39214df8424edc073a18f).